docs/source/dataset_script.mdx

```diff
@@ -270,85 +270,27 @@ def _generate_examples(self, filepath):
 ## Testing data and checksum metadata
 
 We strongly recommend adding testing data and checksum metadata to your dataset to verify and test its behavior. This ensures the generated dataset matches your expectations.
-Testing data and checksum metadata are mandatory for datasets stored in the GitHub repository of the 🤗 Datasets library.
+Testing data and checksum metadata are mandatory for datasets stored in the 🤗 Datasets GitHub repository.
 
 <Tip warning={true}>
 
-Make sure you run all of the following commands **from the root** of your local `datasets` repository.
+Make sure you run the following command **from the root** of your local `datasets` repository.
 
 </Tip>
 
-### Dataset metadata
-
-1. Run the following command to create the metadata file, `dataset_infos.json`. This will also test your new dataset loading script and make sure it works correctly.
+The `dataset_infos.json` file stores the dataset metadata, including the data file checksums, the number of examples required to confirm the dataset was correctly generated, and information about the dataset like its `features`. Run the following command to create `dataset_infos.json` and test your new dataset loading script to make sure it works correctly:
 
 ```
 datasets-cli test path/to/<your-dataset-folder> --save_infos --all_configs
```

Comment:
Here I would suggest path to loading script rather than dataset folder:
- Both work exactly the same
- But it is highly probable that the user is already inside the dataset folder: therefore there is no need to go up a directory; they can run this command directly from inside their dataset directory
```suggestion
datasets-cli test path/to/<your-dataset-loading-script> --save_infos --all_configs
```